### PR TITLE
add the rest of the eth methods

### LINKED ISSUE #495: Audit JSON-RPC
It would be great if we could create a doc with all the existing namespaces and their corresponding methods (such as the table here https://github.com/cosmos/ethermint/issues/508) and annotate if they apply to Etheremint and if they are implemented or not (yet).

### PR DESCRIPTION
**Pull request by J-Thompson12**
Ref #534
Closes: #495 

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Review by fedekunze — APPROVED** [review]
Thanks @J-Thompson12! I was thinking we could also add a `Notes` column like the one on this issue https://github.com/cosmos/ethermint/issues/508 to explain why the method doesn't apply

**Comment by J-Thompson12** [comment]
Thats not a bad idea. I would probably just copy the about the call for the rest of them.

File: docs/basics/json_rpc.md [hunk 1]
```diff
@@ -643,6 +733,164 @@ Unsubscribe from an event using the subscription id
 {"jsonrpc":"2.0","result":true,"id":1}
 ```
 
+### personal_importRawKey
```
Comment (fedekunze) [inline docs/basics/json_rpc.md:738]:
```suggestion
## Personal Methods

### personal_importRawKey
```

**Comment by J-Thompson12** [comment]
Thanks I missed that line

**Pull request merged**